nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-processors/src/test/java/org/apache/nifi/processors/cassandra/PutCassandraRecordTest.java

```diff
@@ -120,6 +120,27 @@ public void testSimplePut() throws InitializationException {
         testRunner.assertAllFlowFilesTransferred(PutCassandraRecord.REL_SUCCESS, 1);
     }
 
+    @Test
+    public void testSimpleUpdate() throws InitializationException {
```

Comment:
Is it possible to add unit tests for the other update methods? I know the whole "server" is mocked out, but might be nice to test non-happy-paths such as non-Long fields attempting to be incremented.